tool/release_notes/release_notes.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Can we change this to a YAML file so it is easier to edit?

Reply:
This might need some more discussion.
For V1 of this i have it as json since I have serialized classes that we can use for validation and doing fancy stuff in the github workflows